examples/compare_CPMG_echo.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from pyspecdata import find_file, figlist_var
+from pyspecProcScripts import lookup_table, select_pathway
+
+with figlist_var() as fl:
+    for thisfile, exp_type, nodename, thislabel in [
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_13",
+            "CPMG simple cyc",
+        ),
+        (
+            "240702_13p5mM_TEMPOL_CPMG.h5",
+            "ODNP_NMR_comp/CPMG",
+            "CPMG_16",
+            "CPMG large cyc",
+        ),
+        (
+            "240702_13p5mM_TEMPOL_echo.h5",
+            "ODNP_NMR_comp/Echoes",
+            "echo_7",
+            "echo large cyc",
+        ),
+    ]:
+        thisd = find_file(
+            thisfile, exp_type=exp_type, expno=nodename, lookup=lookup_table
+        )
+        thisd.squeeze()
+        fl.next("raw data for %s" % thislabel)
+        rows = np.prod([thisd.shape[j] for j in thisd.dimlabels[:-1]])
+        if rows < 500:
+            fl.image(thisd)
+        else:
+            fl.image(thisd, interpolation="bilinear")
+        thisd.ift("t2")
+        fl.next("abs(t domain) comparison")
+        thisd = select_pathway(thisd, thisd.get_prop("coherence_pathway"))
```

Comment:
But you need to grab alternating pathways for alternating echoes for the 4x4x4 cpmg. It's not just one pathway.

If you're using the data with the incorrectly assigned axis, you will also want to divide by 4. 

I would recommend that the easy solution to this is to pass an extra argument in the list of tuples at the top, which is a function that either does what you are doing here, or does what you need to do for the 4x4x4 cpmg

Reply:
okay I added odd and even which I think is what we want to do?